Add test scaffolding for node tasks

damccorm opened the pull request:
Adds a basic  hello world test to each test suite which is missing tests. Goal is to make #11731 a little bit easier

ShreyasRmsft commented:
RunDistributedTestsV1 is deprecated actually, we can remove the code from master.  The jmeter and load test tasks are also on the path to deprecation by early next year, do we want to still add the test scaffholding?

damccorm commented:
> RunDistributedTestsV1 is deprecated actually, we can remove the code from master. The jmeter and load test tasks are also on the path to deprecation by early next year, do we want to still add the test scaffholding?

I removed RunDistributedTests from this PR, added #11758 to get rid of it entirely. I'd say lets leave the other ones. Worst case we don't do anything with them.
